test/e2e/data/kustomize/common-patches/containerd-kct.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# This ensures that containerd is installed and configures the system for kubeadm.
+- op: add
+  path: /spec/template/spec/preKubeadmCommands
+  value:
+  - /usr/local/bin/ci-pre-kubeadm.sh
+- op: add
+  path: /spec/template/spec/files/-
+  value:
+    content: |
+      #!/bin/bash
+      USE_CI_ARTIFACTS=${USE_CI_ARTIFACTS:=false}
+      if [ ! "${USE_CI_ARTIFACTS}" = true ]; then
+        echo "No CI Artifacts installation, exiting"
+        exit 0
+      fi
+      echo "Installing containerd"
+      apt-get install -y containerd
+    owner: root:root
+    path: /usr/local/bin/ci-pre-kubeadm.sh
+    permissions: "0750"
+- op: add
+  path: /spec/template/spec/files/-
+  value:
+    content: |
+      br_netfilter
+    path: /etc/modules-load.d/crio-net.conf
```

Comment:
looks we are using containerd but here is crio
is it on purpose or naming convention is this?

Reply:
We should change this to containerd. It was a suggestion from my own environment where I use crio.

Reply:
I changed it to `k8s.conf`, is that alright?